internal/config/config.go

```diff
@@ -78,6 +78,26 @@ type ArtifactDownload struct {
 	Directory string   `yaml:"directory,omitempty" json:"directory"`
 }
 
+type sendStatus string
```

Comment:
Hm, maybe we can re-use the conditions as defined by `When`? (https://github.com/saucelabs/saucectl/blob/main/internal/config/config.go#L64)

Reply:
Good idea! I fixed this issue.